src/components/index.js

```diff
@@ -2,5 +2,6 @@ import Home from './Home';
 import About from './About';
 import Contact from './Contact';
 import Navigation from './Navigation';
+import {BrowserRouter as Router} from "react-router-dom";
```

Comment:
Its not necessary to import BrowserRouter into the component if you're not going to utilize it.